packages/datetime/test/dateInputTests.tsx

```diff
@@ -14,6 +14,14 @@ import { padWithZeroes } from "../src/common/utils";
 import { Classes, DateInput } from "../src/index";
 
 describe("<DateInput>", () => {
+    it("handles null inputs without crashing", () => {
+        mount(<DateInput value={null} />);
```

Comment:
i agree on the clarity @cmslewis. it's not obvious that we didn't simply forget to finish the test case here.